scene-lib/src/scenelib/annotations/field/AnnotationAFT.java

```diff
@@ -49,9 +51,36 @@ public  String toString() {
     /**
      * {@inheritDoc}
      */
+    // TODO: reduce duplication with annotator.specification.IndexFileSpecification.getElementAnnotations(AElement)
     @Override
     public String format(Object o) {
-        return o.toString();
+        Annotation anno = (Annotation) o;
+        StringBuilder result = new StringBuilder();
+        // TODO: figure out how to consider abbreviated annotation names.
+        // See annotator.find.AnnotationInsertion.getText(boolean, boolean)
+        result.append("@" + annotationDef.name);
+        if (anno.fieldValues.size() == 1 && anno.fieldValues.containsKey("value")) {
+            AnnotationFieldType fieldType = annotationDef.fieldTypes.get("value");
+            result.append('(');
+            result.append(fieldType.format(anno.fieldValues.get(anno.fieldValues.get("value"))));
+            result.append(')');
+        } else if (anno.fieldValues.size() > 0) {
+            result.append('(');
+            boolean notfirst = false;
```

Comment:
Using a positive variable such as `first` is usually easier to understand than a negated one.